examples/unfilter_testbench.rs

```diff
@@ -0,0 +1,111 @@
+// Microbenchmark for all the various filters
```

Comment:
We already have a benchmark for unfiltering, https://github.com/image-rs/image-png/blob/master/benches/unfilter.rs

I think it would be better to unify them so that both are runnable through `cargo bench`

Reply:
I think I should remove this before merging: it's only really useful for the micro-architectural simulation (since Criterion.rs is not designed to run in such a restricted environment), but it's here in the PR for context and if you wish to request clarifications and changes.